tools/nl/embeddings/README.md

```diff
@@ -54,3 +54,12 @@ To allow the `gspread` library access to the google sheets above, you will need
 As of Feb 2023, you can download the gspread-python-app credentials [found here](https://pantheon.corp.google.com/apis/credentials/oauthclient/878764285063-2tqmvvstv8k8cdl7ougccd7ptpnat8d5.apps.googleusercontent.com?project=datcom-204919) to `~/.config/gspread/credentials.json`.
 
 
+# Fine Tuning the Model
+
+To fine tune an existing  `sentence_transform` model, do the following:
```

Comment:
Is there a reference you followed for fine-tuning embeddings?  That might be useful to include here.

Reply:
Added a reference